packages/diagrams/frontend/sirius-components-diagrams-reactflow/src/renderer/RectangularNode.tsx

```diff
@@ -48,8 +56,10 @@ export const RectangularNode = memo(({ data, isConnectable, id, selected }: Node
   return (
     <>
       <NodeResizer color="var(--blue-lagoon)" isVisible={selected} />
-      <div style={rectangularNodeStyle(data.style, selected)}>
-        <Label label={data.label} />
+      <div style={rectangularNodeStyle(data.style, selected, data.fadded)}>
+        <div data-id={data.label.id} style={labelStyle(data.label.style, data.fadded)}>
```

Comment:
Update the label style directly in the Label component. Add a props to the Label component to give the fade state.